src/ui/public/vislib/visualizations/point_series/_point_seri.js

```diff
@@ -0,0 +1,87 @@
+import _ from 'lodash';
+import errors from 'ui/errors';
+
+export default function PointSeriProvider(Private) {
+
+  class PointSeri {
```

Comment:
Is this name a typo? I don't think "seri" is a word I've ever seen before (and I couldn't find any definition for it online), so the name doesn't help me understand the role of this class. Would just using "series" make sense for the role of this class?

This suggestion applies to the `seri_types.js` file too. That should be `series_types` I think.